Jenkinsfile

```diff
@@ -258,7 +258,8 @@ spec:
                                         }
                                     }
                                 }
-                                stash includes: "${toStashDist}", name: 'mac2'
+                                stash includes: "${toStashDist-mac}", name: 'mac2'
```

Comment:
was defined at the top of the Jenkinsfile like this: `toStashDist = "${distFolder}/**"`
We need to define additional patterns for `toStashDist-mac` and `toStashDist-mac-arm`

Reply:
I think the subdirectories are
applications/electron/dist/mac-arm64 
applications/electron/dist/mac-x64

Reply:
Updated :+1:

Reply:
As far as I can see, `toStashDist` is still used in the Jenkinsfile, so it can't be deleted.

And I'm not completely sure about this, but I believe we also need the `/**` at the end of `toStashDist-mac` and `toStashDist-mac-arm`. I'm never quite sure with glob patterns though.